pkg/util/legacy/legacy.go

```diff
@@ -1,4 +1,4 @@
-package kubeobjects
+package legacy
```

Comment:
@waodim can we find better name or remove/refactor it?

Cc @0sewa0

Reply:
I renamed it now from legacy to oneagentapm, as in this file there is only a function regarding checking for oneagentapms. I thought legacy as a name might be a little bit overkill. What do you guys think? done in https://github.com/Dynatrace/dynatrace-operator/pull/2336/commits/b58b302251d3905a66492b1b0993d5ee3d7f2669